.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,38 @@
+name: 🌱 New Feature Request
+description: This template is for requesting a new feature be added.
+labels: [feature request]
+body:
+- type: textarea
+  attributes:
+    label: Increasing Access 
+    description: How would this new feature help [increase access](https://github.com/processing/p5.js/blob/main/contributor_docs/access.md) to p5.js?
```

I like the idea behind this being required, but I'd also like to give people an out if they don't know. I've seen folks submit an idea with this question blank and have others chime in with their own illuminating ideas. Maybe we could do something like this?

```suggestion
    description: How would this new feature help [increase access](https://github.com/processing/p5.js/blob/main/contributor_docs/access.md) to p5.js? (If you're not sure, you can type "Unsure" here and let others from the community offer their thoughts.)
```